.github/workflows/issue-labeler.yml

```diff
@@ -0,0 +1,19 @@
+name: Issue Labeler
+on:
+  issues:
+    types: [opened, edited]
+
+permissions:
+  issues: write
+  contents: read
+
+jobs:
+  triage:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: github/issue-labeler@v3.1 
+      with:
+        configuration-path: .github/issue-labeler.yml
+        not-before: 2020-01-15T02:54:32Z
+        enable-versioned-regex: 0
```

Comment:
What does this do?

Reply:
Best I can see is https://github.com/github/issue-labeler?tab=readme-ov-file#example-using-versioned-issue-templates, but I'm also not sure.  I don't think it applies to what we're doing, but I also don't know whether we have to explicitly state that it's turned off.